index.js

```diff
@@ -10,16 +10,13 @@ function glslifyWebpackLoader(source) {
   this.callback = this.async()
   this.cacheable(true)
 
-  glslify.bundle(source, {
-    inline: true,
-    basedir: basedir
-  }, function(err, src, files) {
-    if (files) {
-      files.forEach(function(file) {
```

Comment:
Is this accomplished implicitly by `glslify()` (as opposed to `glslify.bundle`) or otherwise no longer needed?

Reply:
I suppose so, according to their README: 

> var src = glsl(shaderSource, opts)
> 
> Compile a shader using an inline shader string or a file name.
> 
> These are convencience methods provided that call glsl.compile() or glsl.file() accordingly. These methods are also provided for backwards compatibility with the previous < 6 interface.

Reply:
Sounds good. Just checking. I'll only register the possibility that some dependency registration might have been lost that could interfere with live reloading or something.

Regarding `glslify()`, I might lean toward the non-convenience methods (e.g. `glslify.file()`) since I've had issues in the past with the convenience method making some inferences (string without newline = filename, string with filename = shader source) that were a bit surprising.

Reply:
Want me to revert the changes here then?

Reply:
I think fine as-is, but maybe just use `glslify.file()` (I think that's the one?) instead of `glslify()` since it's more explicit and doesn't rely on convenience/inference.

Reply:
Done. It's `glslify.compile()` as `source` arg refers to the content from the file that passed the regex test within the config.